addons/default/visiosoft/profile-module/src/Events/UserActivatedByMail.php

```diff
@@ -14,4 +14,9 @@ public function __construct($user)
     {
         $this->user = $user;
     }
+
+    public function getUser()
+    {
+        return $this->user;
+    }
```

Comment:
make the $user variable protected, or else there is no point in using getUser(), and if you want you can remove 'use Dispatchable, InteractsWithSockets, SerializesModels;'

Reply:
and please change it in the bonus module in this file addons/default/visiosoft/bonus-module/src/Listeners/CreateEmailActivationBonus.php:12

Reply:
I didn't change it because I thought it might spoil other places.